pkg/apicapi/apicapi.go

```diff
@@ -44,6 +44,13 @@ import (
 // RefreshInterval is set to 0
 const defaultConnectionRefresh = 30 * time.Second
 
+// ApicVersion - This global variable to be used when dealing with version-
+// dependencies during APIC interaction. It gets filled with actual version
+// as part of runConn()
+var (
+	ApicVersion = 3.1
```

Comment:
If we are initializing here, it should probably be 3.2 since we don't support 3.1.